Tests/Card Tests/Utilities/ThrottlerTests.swift

```diff
@@ -29,12 +29,11 @@ class ThrottlerTests: XCTestCase {
                 }
             }
             
-            wait(for: .milliseconds(100))
+            wait(for: .milliseconds(5))
```

Comment:
Quite a significant change here, are these numbers mostly come from trial and error (respecting CI env as well)?

Reply:
This test is actually testing throttling/debouncing behavior - as I reduced the `minimumDelay` i was able to also reduce the time between the calls to `throttle`

Reply:
Oh, I see. Magic number can be tricky. Maybe for better readability we can extract those into a nice named constants in the future